modules/cluster-logging-collector-limits.adoc

```diff
@@ -19,23 +19,20 @@ $ oc -n openshift-logging edit ClusterLogging instance
 +
 [source,yaml]
 ----
-apiVersion: "logging.openshift.io/v1"
-kind: "ClusterLogging"
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogging
 metadata:
-  name: "instance"
+  name: instance
   namespace: openshift-logging
-
-...
```

Comment:
I think this ellipses is still showing up here: 
![image](https://github.com/openshift/openshift-docs/assets/77501157/70f4a190-ac9b-450b-a6b5-bea0c6713477)

and I don't notice the one that you added on line 36 in the preview.

Reply:
Pretty sure it looks OK in the preview: https://69107--ocpdocs-pr.netlify.app/openshift-enterprise/latest/logging/log_collection_forwarding/cluster-logging-collector#cluster-logging-collector-limits_cluster-logging-collector

I think you may be confusing it with https://69107--ocpdocs-pr.netlify.app/openshift-enterprise/latest/logging/config/cluster-logging-memory#cluster-logging-memory-limits_cluster-logging-memory which isn't part of this PR 🙂